src/Nox.Types.EntityFramework/Configurations/NoxDtoDatabaseConfigurator.cs

```diff
@@ -43,6 +45,43 @@ private static void ConfigureAttributes(NoxSolutionCodeGeneratorState codeGenera
             }
         }
     }
+
+    private static void ConfigureRelations(NoxSolutionCodeGeneratorState codeGeneratorState, IEntityBuilder builder, Entity entity)
```

Comment:
We can discuss this with Ricardo, not sure if this is an optimal solution, because it actually uses different fields and will require a lot of params, also we may need to change them independently